### ISSUE TITLE
Missing passepartoutkit-source Repository - Build Failure in Passepartout Project

### ISSUE THREAD

**Issue by gillus123** [passepartoutvpn/passepartout#793]
Hello,

Thanks for this open source VPN client, I wanted to recompile the iOS App but got some issues.

Description:

When attempting to build the Passepartout project, the build fails due to an inaccessible or missing `passepartoutkit-source` repository. The error message read:
<img width="646" alt="image" src="https://github.com/user-attachments/assets/342bad92-7e3f-4106-ac87-b53533edb202">

> The remote repository could not be accessed. Make sure a valid repository exists at the specified location and that the correct credentials have been supplied.

Steps Taken:

Checked the list of repositories under passepartoutvpn organization but could not locate `passepartoutkit-source`.
Request: Could you please clarify if passepartoutkit-source is available, or did I miss something?

BR

**Comment by gillus123** [comment]
When trying to clone the passepartoutkit-source repository, I encountered an authentication failure:

> Cloning into 'passepartoutkit-source'...
> remote: Invalid username or password.
> fatal: Authentication failed for 'https://github.com/passepartoutvpn/passepartoutkit-source.git/'

So the repository is private.

**Comment by gillus123** [comment]
OK, I got my answer here:
https://github.com/passepartoutvpn/passepartout/issues/504
So `passepartoutkit-source` is not open source, it seems however that `tunnelkit` it back as open source

**Comment by keeshux** [comment]
Correct, PassepartoutKit is the new backing library going forward, and TunnelKit is discontinued. PassepartoutKit will be available later for development purposes, maybe under a different name.

**Comment by gillus123** [comment]
So I can still compile an old version of the App that relied on Tunnelkit for educational purpose? If yes can you point me out to a good version to use with Tunnelkit?
